tests/config/test_data.config

```diff
@@ -115,13 +115,24 @@ params {
                 transcriptome_fasta                            = "${test_data_dir}/genomics/homo_sapiens/genome/transcriptome.fasta"
                 genome2_fasta                                  = "${test_data_dir}/genomics/homo_sapiens/genome/genome2.fasta"
 		        genome_chain_gz                                = "${test_data_dir}/genomics/homo_sapiens/genome/genome.chain.gz"
+                genome_21_fasta                                = "${test_data_dir}/genomics/homo_sapiens/genome/chr21/sequence/genome.fasta"
```

Comment:
aren't you already adding the keys in one of the other PRs? I guess that would be solved when merging and uipdating the branches one after another

Reply:
yeah, ApplyVQSR adds the same keys as this, will need to update one after the other merges, but this way let me get all the PRs ready in tandem and changes should be minimized.

Reply:
With ApplyVQSR merged, this PR no longer makes any changes to the test data config, so this should be all sorted now :)